uber/templates/regform.html

```diff
@@ -541,7 +541,7 @@ <h4 class="h4-inline">
         <input type="text" name="cellphone" id="cellphone" value="{{ attendee.cellphone }}" class="form-control" placeholder="A phone number we can use to contact you during the event">
     </div>
     {% if not attendee.is_dealer or c.PAGE_PATH == '/registration/form' %}
-        <div class="checkbox col-sm-6">
+        <div class="checkbox col-sm-6 col-sm-offset-2">
```

Comment:
This change is 100% not related to this issue. But it's such a small tweak I decided to fit it in here while I was looking at it.